src/components/AnimatedStep/index.tsx

```diff
@@ -18,18 +18,19 @@ type AnimatedStepProps = ChildrenProps & {
     onAnimationEnd: () => void;
 };
 
-function getAnimationStyle(direction: AnimationDirection) {
-    let transitionValue;
+function AnimatedStep({onAnimationEnd, direction = CONST.ANIMATION_DIRECTION.IN, style = [], children}: AnimatedStepProps) {
+    const styles = useThemeStyles();
 
-    if (direction === 'in') {
-        transitionValue = CONST.ANIMATED_TRANSITION_FROM_VALUE;
-    } else {
-        transitionValue = -CONST.ANIMATED_TRANSITION_FROM_VALUE;
-    }
-    return styles.makeSlideInTranslation('translateX', transitionValue);
-}
+    const animationStyle = useMemo(() => {
+        let transitionValue;
+        if (direction === 'in') {
+            transitionValue = CONST.ANIMATED_TRANSITION_FROM_VALUE;
+        } else {
+            transitionValue = -CONST.ANIMATED_TRANSITION_FROM_VALUE;
+        }
```

Comment:
nit: please use the ternary operator in this case. Is simple, no need to declare a variable and rewrite it later